docs/apv.md

```diff
@@ -288,7 +288,7 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
 
 ### Artikel 32 - Crimineel gedrag jegens ambulancemedewerkers en andere overheidsmedewerkers
 
-1. Het is ten strengste verboden om een ambulancemedewerker te ontvoeren.
+1. Het is ten strengste verboden om een ambulancemedewerker te ontvoeren, te gijzelen of negatief te bejegenen. Onder negatief bejegenen valt onder andere het beledigen van ambulancemedewerkers en het niet meewerken met de roleplay van een ambulancier.
```

Comment:
is het niet slim om woorden te gebruiken die iedereen gewoon begrijpt?

Reply:
Aangepast